lib/backend/pgbk/background.go

```diff
@@ -147,6 +146,16 @@ func (b *Backend) runChangeFeed(ctx context.Context) error {
 		b.log.WithError(err).Debug("Failed to silence log messages for change feed session.")
 	}
 
+	// this can be useful if we're some sort of admin but we haven't gotten the
+	// REPLICATION attribute yet
+	// HACK(espadolini): ALTER ROLE CURRENT_USER REPLICATION just crashes postgres on Azure
+	if _, err := conn.Exec(ctx,
+		fmt.Sprintf("ALTER ROLE \"%v\" REPLICATION", poolConfig.ConnConfig.User),
```

Comment:
Any chances for the `User` to be empty?

Reply:
It's legal to have an empty username (just very ill-advised); here it'll just result in a harmless `ERROR:  zero-length delimited identifier at or near """"`. If someone really wants to use the default `postgres` user by running a blank one then they'll just have to deal with not having Teleport enable replication automatically.